legacy/emergency-freeze/20250604_084529/continuity-states/mcp-continuity-service/src/services/applescript_service.py

```
@@ -0,0 +1,324 @@
+"""
+AppleScript Service - Integration with macOS applications and system
+Captures system context and automates macOS workflows
+"""
+
+import asyncio
+import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)